src/Codeception/Lib/Console/Message.php

```diff
@@ -32,7 +33,7 @@ public function style($name)
 
     public function width($length, $char = ' ')
     {
-        $message_length = strlen(strip_tags($this->message));
+        $message_length = Multibyte::strwidth(strip_tags($this->message));
```

Comment:
mb_strwidth is return width of string. multibyte characters are usually twice the width of single byte characters.